### ISSUE TITLE
Question about dataset

### ISSUE THREAD

**Issue by wangchen1801** [GuangxingHan/FCT#9]
Dear author, thank you very much for your work. I have a question about validation dataset. The val dataset used here is coco_2014_val, it has 40000 images and takes forever to do the inference. I am wondering if I can use coco_2017_val as val dataset. Could you please help me clear up my confusion? Thanks!

**Comment by GuangxingHan** [comment]
Please follow the instructions [here](https://github.com/ucbdrive/few-shot-object-detection/blob/master/datasets/README.md#coco) to build up the MSCOCO dataset.

Actually, we do not use the original 40000 coco_2014_val images for testing. All but 5k images from the train/val sets are used for training, while 5k images from the val set are used as the test set. trainvalno5k.json denotes the training set and 5k.json is the test set.
